api/envoy/config/endpoint/v3/endpoint_components.proto

```diff
@@ -104,9 +104,9 @@ message LbEndpoint {
   // of the weights of all endpoints in the endpoint's locality to produce a
   // percentage of traffic for the endpoint. This percentage is then further
   // weighted by the endpoint's locality's load balancing weight from
-  // LocalityLbEndpoints. If unspecified, each host is presumed to have equal
-  // weight in a locality. The sum of the weights of all endpoints in the
-  // endpoint's locality must not exceed uint32_t maximal value (4294967295).
+  // LocalityLbEndpoints. If unspecified or 0, will be treated as 1. The sum
```

Comment:
I think that the PGV constraint should prevent it from being 0, but don't feel strongly against explicitly adding the case of 0. WDYT?